pkg/operatorserver/operator_server.go

```diff
@@ -0,0 +1,24 @@
+package operator_server
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"io"
+	"net/http"
+
+	"github.com/sirupsen/logrus"
+)
+
+// Run runs a simple web server that replies 200.
+func Run(log *logrus.Entry) {
```

Comment:
can we put this in the controller? it's very small and we can start it from SetupWithManager()

Reply:
we can put it there I guess, but it is not a controller essentially. Would you prefer it to be under the controllers?

Reply:
Setting this webserver as a controller doesn't seem to be a better option. Controllers (specifically Builder methods https://pkg.go.dev/sigs.k8s.io/controller-runtime/pkg/builder) expect to work with k8s objects and do operations like watch, reconcile etc. To overcome it for example we must implement `Reconcile` method, even if this is a noop in this case, I don't like it.